mandatory/1-syntax-errors.js

```diff
@@ -13,7 +13,7 @@ console.log(introduceMe("Sonjide", 27));
 function getTotal(a, b) {
   total = a + b;
   // return "The total is total";
-  return total;
+  return "The total is 28";
 }
```

Comment:
I was doing like this before and showed me an error. I changed it like this:
function getTotal(a, b) {
  total = a + b;
  return `The total is ${total}`;

Reply:
Thank you dear.